metrics.ini

```diff
@@ -677,14 +677,20 @@ LineGameplayExtras="lua,CustomOptionRow('GameplayExtras')"
 #    unclear from a usability standpoint.
 LineGameplayExtrasB="lua,CustomOptionRow('GameplayExtrasB')"
 
-LineErrorBar="lua,CustomOptionRow('ErrorBar')"
-LineErrorBarOptions="lua,CustomOptionRow('ErrorBarOptions')"
-LineMeasureCounterOptions="lua,CustomOptionRow('MeasureCounterOptions')"
-LineMeasureCounter="lua,CustomOptionRow('MeasureCounter')"
+## These OptionRows appear on [ScreenPlayerOptions2] and [ScreenAttackMenu],
+## define them here anyway so that each of those screens can fallback on this one
+LineTurn="list,Turn"
+LineScroll="list,Scroll"
+LineLifeMeterType="lua,CustomOptionRow('LifeMeterType')"
 LineDataVisualizations="lua,CustomOptionRow('DataVisualizations')"
 LineTargetScore="lua,CustomOptionRow('TargetScore')"
 LineActionOnMissedTarget="lua,CustomOptionRow('ActionOnMissedTarget')"
-LineLifeMeterType="lua,CustomOptionRow('LifeMeterType')"
+LineErrorBar="lua,CustomOptionRow('ErrorBar')"
+LineErrorBarOptions="lua,CustomOptionRow('ErrorBarOptions')"
+LineMeasureCounter="lua,CustomOptionRow('MeasureCounter')"
+LineMeasureCounterOptions="lua,CustomOptionRow('MeasureCounterOptions')"
+LineTimingWindows="lua,CustomOptionRow('TimingWindows')"
+LineFaPlus="lua,CustomOptionRow('FaPlus')"

```

Comment:
I think this should be in the list for the first page.

Reply:
Done